avocado/core/job.py

```diff
@@ -109,6 +110,15 @@ def __init__(self, args=None):
         self.result = result.Result(self)
         self.sysinfo = None
         self.timeout = getattr(self.args, 'job_timeout', 0)
+        #: The time at which the job has started or `-1` if it has not been
+        #: started by means of the `run()` method.
+        self.time_start = -1
```

Comment:
Updating with the information I remember about the discussion we had (maybe IRC or voice).  This uses `-1` to be consistent with `avocado.core.test.Test`:

```python
    #: (unix) time when the test started (could be forced from test)
    time_start = -1
    #: (unix) time when the test finished (could be forced from test)
    time_end = -1
    #: duration of the test execution (always recalculated from time_end -
    #: time_start
    time_elapsed = -1
```